config/default.conf

```diff
@@ -1,42 +1,63 @@
 [global]
 
-server.socket_host: '0.0.0.0'
-server.socket_port: 81
-
-response.headers.server: "Nothing to see here ... move along :)"
-
-tools.jinja.on: False
-tools.caching.on: False
-tools.isAuthorised.on: False
-
-log.access_file : None
-log.error_file  : 'logs/error.log'
-
-
-email: "EMAIL"
-
-#headnode: "https://HEADNODE/RPC2"
-
-# Sets default values for users set to unlimited
-unlimited-vm: "Unlimited"
-unlimited-cpu: 64
-unlimited-mem: 61440
-unlimited-sys: 61440
-
-auth.session_expire_secs: 86400
-auth.ldap_address: "ldap://LDAP"
-auth.ldap_basedn: ""
-
-goodwords: "/usr/share/dict/words"
-# you must provide this file yourself
-badwords: "/var/www/frontend/badwords"
-
-# websockify config for NoVNC
-wshostname: "HOSTNAME"
-wsport: 29876
-#wscert: "/etc/httpd/ssl/CERT"
-#wskey: "/etc/httpd/ssl/KEY"
-wstokendir: "/var/www/frontend/tokens/"
-
-# set to True if using Apache
-wsgi_enabled: True
+# Auth
+auth.ldap_address = "ldap://LDAP"
+auth.ldap_basedn = "dc=FED,dc=CCLRC,dc=AC,dc=UK"
+auth.session_expire_secs = 86400
+
+# CherryPy Server
+server.socket_host = "0.0.0.0"
+server.socket_port = 8080
+
+# CherryPy Tools
+tools.caching.on = False
+tools.isAuthorised.on = False
+tools.jinja.on = False
+tools.sessions.on = True
+tools.sessions.storage_type = 'File'
+tools.sessions.storage_path = 'sessions'
+tools.sessions.timeout = 60
+
+# Logs
+log.access_file = "logs/access.log"
+log.error_file = "logs/error.log"
+
+# VM names
+goodwords = "/usr/share/dict/words"
+badwords = "/var/www/frontend/assets/badwords" # You must provide this file yourself
+
+# Websockify
+wshostname = "HOSTNAME"
+wsport = 6080
+#wscert = "/etc/httpd/ssl/CERT"
+#wskey = "/etc/httpd/ssl/KEY"
+wstokendir = "/var/www/frontend/tokens/"
+
+# Wsgi
+wsgi_enabled = True
+
+# Platform
+cloudPlatform = "openstack"
+
+# OpenNebula
+headnode = "XMLRPC_API_URL"
+unlimited-cpu = 64
+unlimited-mem = 102400
+unlimited-sys = 102400
+unlimited-vm = "Unlimited"
+
+# OpenStack
+keystone = "KEYSTONE_URL"
+novaVersion = "2.1"
+openstack_host = "OPENSTACK_URL"
+openstack_default_domain = "USER_DOMAIN"
+vmNetworkLabel = "NETWORK_LABEL"
+securityGroupName = "SEC_GROUP"
+flavorPrefix = "m"
+availabilityZoneName = "AVAILIABILTY_ZONE"
+aqProfiles = "AQUILON_URL/profiles/"
+countLimit = 10
+
+# Misc
+response.headers.server = "Nothing to see here ... move along :)"
```

Comment:
???

Reply:
Are you going to replace this with something similar to the readme?

Reply:
No, that string is the actual value for this key, set by Callum Williams. I have no reason to believe it should be changed.

Reply:
Hmm, ok.